src/flatc.cpp

```diff
@@ -319,6 +319,8 @@ int FlatCompiler::Compile(int argc, const char **argv) {
         opts.force_defaults = true;
       } else if (arg == "--force-empty") {
         opts.set_empty_to_null = false;
+      } else if (arg == "--java_primitive_has_method") {
```

Comment:
can you make all the `_` into `-` to be consistent with the other options?

Reply:
Done.